src/content/docs/browser/new-relic-browser/browser-pro-features/geography-webpage-metrics-location.mdx

```diff
@@ -19,48 +20,48 @@ import browserFilterableGraphs from 'images/browser_screenshot-full_filterable-g
 import browserGeoKeyAttributes from 'images/browser_screenshot-full_key-attributes.png'
 
 
-With New Relic browser monitoring, the **Filterable geos** UI page provides a world view with color-coded performance information about your front-end experience in cities, regions, and countries anywhere around the world. The map shows your user data by region, so you can visualize your traffic and error hotspots alongside device type information. You can also examine critical Core Web Vitals data so that you can prioritize areas around the globe needing attention.
+With New Relic browser monitoring, the **Geography** UI page provides a world view with color-coded performance information about your front-end experience in cities, regions, and countries anywhere around the world. The map shows your user data by region, so you can visualize your traffic and error hotspots alongside device type information. You can also examine critical Core Web Vitals data so that you can prioritize areas around the globe needing attention.
 
-If you work in IT or operations, this level of visual detail is especially useful for making business decisions about peering agreements and CDN usage. For example, if it's unclear which regions are experiencing higher page load times than normal, our **Filterable geos** UI can help you track down the root of the problem.
+If you work in IT or operations, this level of visual detail is especially useful for making business decisions about peering agreements and CDN usage. For example, if it's unclear which regions are experiencing higher page load times than normal, our **Geography** UI can help you track down the root of the problem.
 
 For some tips on how to make the most of the available data, see the [filter examples](#filter).
 
 ## Troubleshoot performance location [#view-data]
 
-Filterable geography functionality requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types). If you have not installed this agent type, you may be able to use the [legacy **Geography** UI page](#legacy-geo).
+The Geography UI functionality requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types). If you have not installed this agent type, you may be able to use the [legacy UI page](#legacy-geo).
```

Comment:
I might have to remove this. Waiting to hear back from PM if this means all users are able to see/use the **Geography** page.